pkg/probes/dummy/dummy.go

```diff
@@ -0,0 +1,38 @@
+package dummy
+
+import (
+	"github.com/openshift/osd-network-verifier/pkg/output"
+)
+
+type DummyProbe struct{}
```

Comment:
Can we rename this to just `Probe`? The package name is `dummy` so external callers will already have to qualify it as `dummy.Probe`, whereas `dummy.DummyProbe` can be a bit redundant.